_posts/01-04-01-Mac-Setup.md

```diff
@@ -4,63 +4,52 @@ isChild: true
 
-MacPorts supports pre-compiled binaries, so you don't need to recompile every
-dependencies from the source tarball files, it saves your life if you don't
-have any package installed on your system.
+MacPorts es compatible con binarios pre-compilados, por tanto no necesitas recompilar cada dependencia de los archivos tarball desde la fuente, ahorrándole futuros problemas si no tiene ningún paquete necesario instalado en su sistema.
 
-At this point, you can install `php54`, `php55`, `php56` or `php70` using the `port install` command, for example:
+En este punto, puedes instalar 'php54', 'php55', 'php56' o 'php70' usando el comando 'port install', por ejemplo:
 
-    sudo port install php56
-    sudo port install php70
+sudo port install php56
+sudo port install php70
 
-And you can run `select` command to switch your active php:
+Y puedes correr el comando 'select' para cambiar tu versión de PHP activa:
 
-    sudo port select --set php php70
+sudo port select --set php php70
 
-### Install PHP via phpbrew
+### Instalar PHP via phpbrew
 
-[phpbrew] is a tool for installing and managing multiple PHP versions. This can be really useful if two different
-applications/projects require different versions of PHP, and you are not using virtual machines.
+[phpbrew] es una herramienta para la instalación y administración de múltiples versiones de PHP. Esto puede ser de mucha utilidad si tienes dos diferentes aplicaciones y/o proyectos que requieran versiones diferentes de PHP, y no este usando máquinas virtuales.
 
-### Install PHP via Liip's binary installer
+### Instalar PHP vía Instalador Binario de Liip
 
-Another popular option is [php-osx.liip.ch] which provides one liner installation methods for versions 5.3 through 7.0.
-It doesn't overwrite the php binaries installed by Apple, but installs everything in a separate location (/usr/local/php5).
+Otra opción muy popular es [php-osx.liip.ch], el cual provee un método de instalación para las versiones PHP desde 5.3 a la 7.0.
+Esto no sobrescribirá los binarios de PHP instalados por Apple, sino que instalará los nuevos en una ubicación separada (/usr/local/php5).
 
-### Compile from Source
+### Compilar desde la Fuente
 
-Another option that gives you control over the version of PHP you install, is to [compile it yourself][mac-compile].
-In that case be sure to have installed either [Xcode][xcode-gcc-substitution] or Apple's substitute
-["Command Line Tools for XCode"] downloadable from Apple's Mac Developer Center.
+Otra opción que le da el control sobre la versión de PHP que va a instalar, es [compilarla usted mismo][mac-compile].
+En ese caso asegúrese de tener instalado [Xcode][xcode-gcc-substitution] o un Sustituto ["Herramientas de Linea de Comandos para XCode"] de Apple que se encuentre disponible para la descarga desde el Centro de Desarrolladores de Apple (Apple's Mac Developer Center).
```

Comment:
Cambiar ["Herramientas de Linea de Comandos para XCode"] por ["Herramientas de Linea de Comandos para XCode"]["Command Line Tools for XCode"]